itest/starboard-operator/suite_test.go

```diff
@@ -45,11 +52,11 @@ func TestStarboardOperator(t *testing.T) {
 	RunSpecs(t, "Starboard Operator")
 }
 
-var _ = BeforeSuite(func(done Done) {
```

Comment:
Getting rid of deprecation warning: Done channel and timeout will be deprecated in Ginkgo v2.